locations/spiders/handelsbanken_gb.py

```diff
@@ -0,0 +1,28 @@
+import scrapy
+
+from locations.categories import Categories
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class HandelsbankenGBSpider(scrapy.Spider):
+    name = "handelsbanken_gb"
+    item_attributes = {"brand": "Handelsbanken", "brand_wikidata": "Q1421630", "extras": Categories.BANK.value}
+    start_urls = ["https://www.handelsbanken.co.uk/rsoia/parg/bu/branches/v3/country/GB/en"]
+
+    def parse(self, response, **kwargs):
+        for location in response.json():
+            item = DictParser.parse(location)
+            item["name"] = "Handelsbanken"
+            item["branch"] = location["name"]
+            if location["branchUrl"]:
+                item["website"] = "https://" + location["branchUrl"]
```

Comment:
The 4 without websites are "Regional Bank ..." I'm not convinced they are real banks, how do you feel about dropping them?

Reply:
Agree - couldn't decide - they do have addresses and coordinates so I wasn't sure.